sa-token-core/src/main/java/cn/dev33/satoken/util/ExcelUtils.java

```diff
@@ -0,0 +1,215 @@
+package cn.dev33.satoken.util;
+
+import jxl.*;
+import jxl.format.*;
+import jxl.write.*;
+import org.springframework.util.StringUtils;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.*;
+import java.util.List;
+
+import static cn.dev33.satoken.SaManager.log;
+
+
+public class ExcelUtils {
+
+	public static InputStream getPathByFileName(String template, String tmpFileName) {
+		File tmpFile = new File(template, tmpFileName);
+		InputStream path = null;
+		//判断文件或文件夹是否存在
+		if (tmpFile.exists()) {
+			try {
+				path = new FileInputStream(tmpFile);
+			} catch (FileNotFoundException e) {
+				log.error("", e);
+			}
+		}
+		return path;
+	}
+
+	/**
+	 * 导出excel，带多sheet
+	 *
+	 * @param wtwb
+	 * @param tip
+	 * @param names
+	 * @param title
+	 * @param index
+	 * @param objects
+	 * @return
+	 * @throws Exception
+	 */
+	public static void exportObjectsManySheet(WritableWorkbook wtwb, String tip,
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/xiejijun/sa-token/pull/1#issuecomment-2304574048>#12e843</a><span> - 08/22/2024, 12:43 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The method declares that it throws a generic Exception, which is not a good practice for specific error handling and can make it difficult for callers to handle errors appropriately.
</div>
<div id="fix">
<b>Fix:</b> Replace the generic Exception with specific exceptions that can occur during Excel operations. Consider creating a custom exception for Excel-related errors if needed.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -41,7 +41,7 @@ public class ExcelUtils {
  	 * @throws Exception
  	 */
  	public static void exportObjectsManySheet(WritableWorkbook wtwb, String tip,
 -											  String[] names, String title, int index, List<String[]> objects) throws Exception {
 +											  String[] names, String title, int index, List<String[]> objects) throws IOException, WriteException {
  		WritableSheet sheet = wtwb.createSheet(title, index);
  		sheet.getSettings().setDefaultColumnWidth(12);
 
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged